### PR TITLE
docs: improved cloudflare docs

### PR DESCRIPTION
**Pull request by arthurfiorette**
### What's added in this PR?

New documentation regarding Cloudflare Integration and how to setup environments

### (Required) Pre-PR/Merge checklist

- [x] I have added/updated our feature to sync with this PR
- [x] I have added an explanation of my changes
- [ ] I have/will run tests, or ask for help to add test

### CODE REVIEW

**Comment by StashBank** [comment]
Sugest have screenshots in dark mode as we have our UI in that mode

**Review by zmzlois — APPROVED** [review]

File: docs/cloud/cloudflare.mdx [hunk 1]
```diff
@@ -64,165 +63,146 @@ Before you move to the next step, you will need two details and one DNS configur
...
 
-**3. Configure your DNS record**
+**3. Configure Your DNS Record**
 
+- Return to your domain’s home page (the **Overview** page should still be open) and click on **DNS** on the left sidebar.
+- Click **Add record** on the _Records_ page and enter the following details:
 
-- Go back to your domain's home page (hope that **Overview** page is still opened at this point) and click on **DNS** on left side panel.
-- Click on **Add record** on _Records_ page and add below details:
+![dns](../public/cf-dns-entry-cname.png)
```
Comment (zmzlois) [inline docs/cloud/cloudflare.mdx:76]: give them a table they can copy paste?

File: docs/cloud/cloudflare.mdx [hunk 2]
```diff
@@ -64,165 +63,146 @@ Before you move to the next step, you will need two details and one DNS configur
...
+<dt>Cloudflare Project Name</dt>
 <dd>
-Obtained from Cloudflare. You will need a valid domain on Cloudlfare.
-</dd>
-<dt>Cloudflare project name</dt>
-<dd>
-  This operation is idempotent, it will create a [cloudflare pages
-  project](https://developers.cloudflare.com/pages) on Cloudflare.
+  This operation is idempotent and will create a [Cloudflare Pages
+  project](https://developers.cloudflare.com/pages).
```
Comment (zmzlois) [inline docs/cloud/cloudflare.mdx:101]:
```suggestion
  project](https://developers.cloudflare.com/pages). This field will become the name of your Cloudflare Pages project.
```

File: docs/cloud/cloudflare.mdx [hunk 3]
```diff
@@ -64,165 +63,146 @@ Before you move to the next step, you will need two details and one DNS configur
...
 </dd>
-<dt>Set integration as default.</dt>
+
+<dt>Set Integration as Default</dt>
 <dd>
-  Make every repository deployed through Zephyr will attempt to deploy through this
-  integration setting. This won't change until you delete this integration or
-  add new integration as default.
+  When set as default, all Zephyr deployments will use this integration until a
+  new one is set.
```
[the remark this answers is not in the document]
Comment (zmzlois) [reply docs/cloud/cloudflare.mdx:107]: new one is vaguely implying "just an integration" or "default integration" and it needs to be default integration to be effective

**Review by zmzlois — CHANGES_REQUESTED** [review]
in general very well written ❤️

**Review by zmzlois — APPROVED** [review]
LGTM

**Pull request merged**